k8s/openshift/byo/baremetal/containerized_openshift_readme.md

```diff
@@ -1,5 +1,475 @@
+## Procedure to run OpenEBS on Multi-Node Cotainerized OpenShift Cluster
+
+This tutorial provides detailed instructions on how to setup a multi-node BYO (Bring-Your-Own-Host) OpenShift Containerized cluster on 
+RHEL 7.5 and run applications on it with OpenEBS storage.
+
+### PRE-REQUISITES
+
+-At least 2 or more RHEL 7.5 hosts (virtual-machines/baremetal/cloud instances) with 3 vCPUs, 16G RAM and 60GB Hard disk.
+-A valid Red Hat subscription
+
+### Attach OpenShift Container Platform Subscription
+
+1.As root on the target machines (both master and node), use subscription-manager to register the systems with Red Hat.
+
+```
+$ subscription-manager register
+```
+
+2.Pull the latest subscription data from RHSM:
+
+```
+$ subscription-manager refresh
+```
+
+3.List the available subscriptions.
+
+```
+$ subscription-manager list --available
+```
+
+4.Find the pool ID that provides OpenShift Container Platform subscription and attach it.
+
+```
+$ subscription-manager attach --pool=<pool_id>
+```
+
+5.Replace the string <pool_id> with the pool ID of the pool that provides OpenShift Container Platform. The pool ID is a long alphanumeric string.
+
+These RHEL systems are now authorized to install OpenShift Container Platform. Now you need to tell the systems from where to get 
+OpenShift Container Platform.
+
+### Set Up Repositories
+On both master and node, use subscription-manager to enable the repositories that are necessary in order to install OpenShift Container 
+Platform. You may have already enabled the first two repositories in this example.
+
+```
+$ subscription-manager repos --enable="rhel-7-server-rpms" \
+    --enable="rhel-7-server-extras-rpms" \
+    --enable="rhel-7-server-ose-3.9-rpms" \
+    --enable="rhel-7-fast-datapath-rpms" \
+    --enable="rhel-7-server-ansible-2.4-rpms"
+```
+
+This command tells your RHEL system that the tools required to install OpenShift Container Platform will be available from these 
+repositories. Now we need the OpenShift Container Platform installer that is based on Ansible.
+
+
+### Install the OpenShift Container Platform Package
+The installer for OpenShift Container Platform is provided by the atomic-openshift-utils package. Install it using yum on both the 
+master and the node, after running yum update.
+
+```
+$ yum -y install wget git net-tools bind-utils iptables-services bridge-utils bash-completion kexec-tools sos psacct
+$ yum -y update
+$ yum -y install atomic-openshift-utils
+$ yum -y install docker
+```
+
+-Functional DNS server, with all hosts configured by appropriate domain names (Ensure nslookup of the hostnames is successful in 
+ resolving the machine's IP addresses).The detail steps can be found in below mentioned link.
```

Comment:
addresses). <space> The detailed steps can be found by going to the following link.